sklearn/metrics/_classification.py

```diff
@@ -1573,7 +1644,7 @@ def precision_recall_fscore_support(
         pred_sum = np.array([pred_sum.sum()])
         true_sum = np.array([true_sum.sum()])
 
-    # Finally, we have all our sufficient statistics. Divide! #
+    # Finally, we have all our sufficient statistics. Divide!
```

Comment:
This can be reverted so a reviewer can focus on the changes that have to zero division = nan

Reply:
Will revert all changes not related to my change. Lmk if you find more